src/commands/PlayCommand.ts

```diff
@@ -24,15 +24,15 @@ export class PlayCommand extends BaseCommand {
         const voiceChannel = message.member!.voice.channel!;
         if (!args[0]) {
             return message.channel.send(
-                createEmbed("warn", `Invalid args, type \`${this.client.config.prefix}help play\` for more info`)
+                createEmbed("warn", `Invalid argument, type \`${this.client.config.prefix}help play\` for more info`)
             );
         }
         const searchString = args.join(" ");
         const url = searchString.replace(/<(.+)>/g, "$1");
 
         if (message.guild?.queue !== null && voiceChannel.id !== message.guild?.queue.voiceChannel?.id) {
             return message.channel.send(
-                createEmbed("warn", `Music on this server is already playing to: **${message.guild?.queue.voiceChannel?.name}** voice channel`)
+                createEmbed("warn", `The music player is already playing to: **${message.guild?.queue.voiceChannel?.name}** voice channel`)
```

Comment:
```suggestion
                createEmbed("warn", `The music player is already playing to **${message.guild?.queue.voiceChannel?.name}** voice channel`)
```
Should we remove the colon?